task_manager/task_manager/task_client.py

```diff
@@ -267,7 +267,7 @@ def __init__(
             )
 
         self._client = self._service_clients[task_specs.task_name]
-        self.cancel_task_timeout = 5.0  # Timeout to wait for task to cancel
+        self.cancel_task_timeout = task_specs.cancel_timeout  # Timeout to wait for task to cancel
```

Comment:
Same here with using the task_specs object

Reply:
Done and fixed some tests after this!